### PR TITLE
Rownin Skeleton and removal of chosing M3 harness

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

From the spriter that brought you #11278, #11479, #10610, #11579, and #11549, they bring up a new armor for any speedy glass cannon marine to BEAT THE XENOMORPH!

The new **VERY LIGHT** armor intended to replace the M3 is called the Rownin Skeleton. For the marines that want to be fast and risk having no armor (see marines using only Jaegar exoskeleton with no Jaegar armor pieces but only use the modules) and have STYLE, they sought out to be glass cannons.

image

This is a picture with modules.
image

Remove M3 harness for marines to use in exchange for Rownin Skeleton.

## Why It's Good For The Game

More bling for the marines that just want to die fast.

Also, the Rownin Skeleton is practically a M3 pattern marine harness that can accept modules, essentially a Jaegar exoskeleton with no Jaegar pieces. This isn't affecting balance at all, but rather giving marines a better visual option for their glass cannon aesthetics.

It is sad that I must remove the option to get M3 harness from marine vendor. M3 and Rownin Skeleton cannot coexist together.

## Changelog

:cl: novaepee & Icon [Credit AGhost] & Code help [Credit HYPER2SNYDER]
add: Rownin Skeleton, a glass cannon armor that can permit modules to be installed. Speed in exchange of armor just like a M3 pattern marine harness or a Jaegar exoskeleton with no Jaegar pieces
del: M3 harness for marines to use in exchange for Rownin Skeleton
/:cl:

### CODE REVIEW

**Comment by FosterGrime-Heymynameslouie** [comment]
the tyr module looks funky

like you have marines shoulder bike safety pads and called it a day

**Comment by Al-1ce** [comment]
The uniforms all sort of look the same but I think any variation is good.

**Review by TiviPlus — CHANGES_REQUESTED** [review]
is this is just harness 2 why does harness exist

**Comment by SilencedMP5A5** [comment]
let us put modules on harness + give harness (and jackets) usable light ranges 2023

**Comment by Losenis** [comment]
>It's under xenonaut parent so synths can't use it

Balls, there goes the 0 armor pure utility armor

File: code/modules/clothing/modular_armor/modular.dm [hunk 1]
```diff
@@ -271,7 +271,18 @@
 /obj/item/clothing/suit/modular/xenonauten/mimir
 	starting_attachments = list(/obj/item/armor_module/module/mimir_environment_protection/mark1)
 
-// Thank Jeff for providing sprites to color flak jacket
+/obj/item/clothing/suit/modular/xenonauten/rownin
+	name = "\improper Rownin Skeleton"
+	desc = "A light armor, if you can even called it that, made by SSgt Rownin for dedicated marines that want to travel light and have agility in exchange of protection. Alt-Click to remove attached items. Use it to toggle the built-in flashlight."
```
[the remark this answers is not in the document]
Comment (novaepee) [reply code/modules/clothing/modular_armor/modular.dm:276]: Made up lore

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 2]
```diff
@@ -1245,6 +1245,8 @@
 			/obj/item/clothing/under/marine/camo/desert = -1,
 			/obj/item/clothing/under/marine/camo/snow = -1,
 			/obj/item/clothing/under/marine/orion_fatigue = -1,
+			/obj/item/clothing/under/marine/red_fatigue = -1,
+			/obj/item/clothing/under/marine/lv_fatigue = -1,
```
[the remark this answers is not in the document]
Comment (novaepee) [reply code/game/objects/machinery/vending/marine_vending.dm:1249]: New fatigues that Ghost provided.

File: code/modules/clothing/under/marine_uniform.dm [hunk 3]
```diff
@@ -85,7 +85,27 @@
...
+	)
+
+/obj/item/clothing/under/marine/lv_fatigue
+	name = "\improper LV-624 fatigues"
+	desc = "Originated from LV-624. Designed for wet, high humid, and jungle environments, they're meant for recon, stealth, and evac operations. They come with a built in cassette player hearable only to the user to help pass time, during any possible long waits. Somewhere, someone is playing 'Fortunate Sons' in the background, and you can smell napalm and Agent Orange in the air..."
+	icon_state = "lv_fatigues"
+	item_state = "lv_fatigues"
+	adjustment_variants = list(
+		"Down" = "_d",
+	)
```
[the remark this answers is not in the document]
Comment (novaepee) [reply code/modules/clothing/under/marine_uniform.dm:108]: New fatigues that Ghost provided.

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**